src/main/native/compression/IntelInflater.cc

```diff
@@ -63,39 +62,36 @@ JNIEXPORT void JNICALL Java_com_intel_gkl_compression_IntelInflater_initNative
   FID_inf_lz_stream = env->GetFieldID(cls, "lz_stream", "J");
   FID_inf_inputBuffer = env->GetFieldID(cls, "inputBuffer", "[B");
   FID_inf_inputBufferLength = env->GetFieldID(cls, "inputBufferLength", "I");
-  FID_inf_endOfStream = env->GetFieldID(cls, "endOfStream", "Z");
   FID_inf_finished = env->GetFieldID(cls, "finished", "Z");
 
 }
 
 JNIEXPORT void JNICALL Java_com_intel_gkl_compression_IntelInflater_resetNative
 (JNIEnv* env, jobject obj, jboolean nowrap) {
 
-    z_stream* lz_stream = (z_stream*)env->GetLongField(obj, FID_inf_lz_stream);
+
+    inflate_state* lz_stream = (inflate_state*)env->GetLongField(obj, FID_inf_lz_stream);
 
     if(lz_stream == 0) {
-      lz_stream = (z_stream*)calloc(1,sizeof(z_stream));
+
+     lz_stream = (inflate_state*)calloc(1,sizeof(inflate_state));
       if ( lz_stream == NULL ) {
         jclass Exception = env->FindClass("java/lang/Exception");
         env->ThrowNew(Exception,"Memory allocation error");
       }
       env->SetLongField(obj, FID_inf_lz_stream, (jlong)lz_stream);
 
-      lz_stream->zalloc = Z_NULL;
-            lz_stream->zfree = Z_NULL;
-            lz_stream->opaque = (voidpf)0;
-            lz_stream->avail_in = 0;
-            lz_stream->next_in = Z_NULL;
+      lz_stream->avail_in = 0;
+      lz_stream->next_in = Z_NULL;
+      lz_stream->avail_out = 0;
+       lz_stream->next_out = Z_NULL;
```

Comment:
does `isal_inflate_init` initialize these? it seems like these four lines should come after `isal_inflate_init`, or maybe they can be removed.